codalab/worker_manager/slurm_batch_worker_manager.py

```diff
@@ -32,6 +32,11 @@ class SlurmBatchWorkerManager(WorkerManager):
 
     @staticmethod
     def add_arguments_to_subparser(subparser):
+        try:
+            user_id = getpass.getuser()
+        except Exception:
+            # Sometimes getpass.getuser() doesn't work.
```

Comment:
Does it not work on the cluster? Should we error out instead?

Reply:
This error always comes up when I try to run a worker manager locally. I had to make the same change to get cs324 worker managers to work.